core/core.go

```diff
@@ -772,6 +803,23 @@ func (c *Core) GetNamespaceKeyValueStoreConfiguration(namespace string) *models.
 			FallbackKvVersion: c.Conf.KeyValueStoreFallbackKvVersion,
 		},
 	}
+}
+
+// GetKeyValueStoreConfiguration - gets key value store configuration
+func (c *Core) GetKeyValueStoreConfiguration() *models.KeyValueStoreConfiguration {
+
+	return &models.KeyValueStoreConfiguration{
+		Type: c.Conf.KeyValueStoreType,
+		Vault: models.VaultKeyValueStoreConfiguration{
```

Comment:
Doesn't this also need to be namespaced?
Just asking to understand.

Reply:
Release Plans can span multiple namespaces - that’s why they are not namespaced and need to be placed in a common place.